Runtime/Plugins/GLTFSerialization/Utilities/GltfRootExtensions.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using System.Collections.Generic;
+using GLTF.Schema;
+
+namespace GLTF.Utilities
+{
+    public static class GltfRootExtensions
+    {
+        public static int[] GetAllNodeIdsWithMaterialId(this GLTFRoot root, int id)
+        {
+            List<int> ids = new List<int>();
+
+            for (int iMeshes = 0; iMeshes < root.Meshes.Count; iMeshes++)
```

Comment:
Can the Meshes array be null? Should we check for that here?
E.g. for a materials-only import file

Reply:
Changed with 427a33216374fd9a6fd7ce222ff7a9e0293f7463